velox/docs/functions/presto/math.rst

```diff
@@ -27,6 +27,17 @@ Mathematical Functions
     verified for performance reasons. Returns ``high`` for all values of ``x``
     when ``low`` is greater than ``high``.
 
+.. function:: cosine_similarity(map(varchar, double), map(varchar, double)) -> double
```

Comment:
Would you document what happens if one or both input maps are empty?

Reply:
Done